.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
-	branch = master
+	url = https://github.com/hannesWell/m2e-core-tests.git
```

Comment:
This part is here by mistake, right?

Reply:
It is here to test my PR in `tesla/m2e-core-tests` before it is merged there.
I stated in my initial comment that this part has to be reverted before this can be merged and that's why I made this a draft.
But since there are test-failures I have to fix them in `tesla/m2e-core-tests`  first it can be merged there.